source/_patterns/01-molecules/components/highlight-item~article-long-authors.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "[Do not use this base pattern directly: see the following variants of this pattern for usage]",
+  "url": "#link-to-article",
+  "authors": "Yongjian Huang, Shashank Bharill, John Kuriyan, Lee R Berger, John Hawks, Bernhard Zipfel, Ming Chao",
```

Comment:
Could this be changed to something like:

```
"authors": "Thisisanexample Ofareallylongauthorline, Todemonstrate Whatitlookslike ... Ifthis Valuewraps",
```

Reply:
Done